src/components/backup-finish.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {useEffect} from 'react';
+
+import {Dimensions} from 'react-native';
+
+import {
+  Button,
+  ButtonVariant,
+  LottieWrap,
+  PopupContainer,
+  Spacer,
+  Text,
+} from '@app/components/ui';
+import {createTheme} from '@app/helpers';
+import {I18N} from '@app/i18n';
+import {HapticEffects, vibrate} from '@app/services/haptic';
+
+const animationSize = Dimensions.get('window').width - 116;
+
+type BackupFinishProps = {
+  onSubmit: () => void;
+};
+
+export const BackupFinish = ({onSubmit = () => {}}: BackupFinishProps) => {
```

Comment:
why set a default parameter if it is required?
better: `export const BackupFinish = ({onSubmit}: BackupFinishProps) => {`